app/Http/Resources/CanvassSummaryItemDetailedResource.php

```diff
@@ -17,12 +17,12 @@ public function toArray(Request $request): array
         $requisitionItem = $this->requisitionSlipItem;
         return [
             'item_id'        => $this->item_id,
-            'quantity'       => $this->quantity,
+            'quantity'       => $this->quantity ?? 0,
```

Comment:
kani and other amounts i wrap og number_format(,2) para ma auto decimal ang display mark